src/Composer/CommandProvider.php

```diff
@@ -10,6 +10,6 @@ class CommandProvider implements CommandProviderCapability {
    * {@inheritDoc}
    */
   public function getCommands() {
-    return [new PatchEnableCommand(), new PatchAddCommand(), new PatchListCommand()];
+    return [new PatchEnableCommand(), new PatchAddCommand(), new PatchRemoveCommand(), new PatchListCommand()];
```

Comment:
Maybe this could be transformed to be multi-line array, to prevent constant changes and reduce conflicts?
(not sure should be part fo this PR, just suggesting as improvement)